index.js

```diff
@@ -4,9 +4,46 @@ const closeNavButton = document.getElementById("close-nav-button");
 const addTaskButton = document.getElementById("add-task-button");
 const taskTitleInput = document.getElementById("task-title-input");
 const taskList = document.getElementById("task-list");
+const taskSortSection = document.getElementById("sort-tasks");
+const sreachInput = document.getElementById("search-input");
+const searchButton = document.getElementById("search-button");
+const timeFilterSection = document.getElementById("filter-by-time-tasks");
 
 const sideMenuContainerWidth = "200px";
-const tasks = [];
+
+let tasks = [
```

Comment:
why `let`?

Reply:
to replace the whole array by allTasks.
but you said it's better to handle it in other way sooooooo I will fix it